ignite/cmd/network_chain_install.go

```diff
@@ -58,9 +59,10 @@ func networkChainInstallHandler(cmd *cobra.Command, args []string) error {
 	}
 	binaryPath := filepath.Join(goenv.Bin(), binaryName)
 
-	fmt.Printf("%s Binary installed\n", clispinner.OK)
-	fmt.Printf("%s Binary's name: %s\n", clispinner.Info, infoColor(binaryName))
-	fmt.Printf("%s Binary's path: %s\n", clispinner.Info, infoColor(binaryPath))
+	session.StopSpinner()
+	session.Printf("%s Binary installed\n", icons.OK)
+	session.Printf("%s Binary's name: %s\n", icons.Info, infoColor(binaryName))
```

Comment:
I'm wondering if we can move infoColor and others out of cmd to my a new package `pkg/cliui/colors`

then we use it as `colors.Info()`